src/components/CustomerAppeal/CustomerAppealScreen.tsx

```diff
@@ -101,6 +101,18 @@ export const CustomerAppealScreen: FunctionComponent<NavigationProps> = ({
               quotaResponse &&
               policy.alert &&
               policyLimit - quotaResponse.quantity >= policyThreshold
+            ) {
+              descriptionAlert = policy.alert.label;
```

Comment:
both if statements (line 100 & line 112) results in `descriptionAlert = policy.alert.label;`, whats the reason for separating them?

Reply:
line 100 checks whether the `threshold` has been reached! however, we don't need that for `PASSPORT` user

Reply:
ahhh, but since both if checks result in 
```
 {
     descriptionAlert = policy.alert.label;
 }
```

do you think we should combine both statements? any value separating them?

Reply:
the clean up is a bit difficult since it will still need to take some arguments 🤔 . The arguments feel quite specific too though.. wdyt?

Reply:
okay i think the changes made should be clear enough!